packages/web/src/components/CategoryResults.tsx

```diff
@@ -51,6 +52,9 @@ const CategoryResults = ({
 
   return (
     <>
+      <Helmet>
+        <title>{t(categoryTranslationKey)} - Upswyng</title>
```

Comment:
Thoughts on creating a utility that centralized appending the "- UpSwyng" to the title? Could create something like `seo.js` in the [utils folder](https://github.com/CodeForBoulder/upswyng/tree/main/packages/web/src/utils) and create a `makePageTitle` that accepts a string as an argument and just returns that appended with ` - UpSwyng`.